Warehouse.Client/warehouse-client/src/app/login/login.component.ts

```diff
@@ -0,0 +1,49 @@
+import {Component, EventEmitter, OnInit, Output} from '@angular/core';
+import {FormControl, FormGroup, Validators} from '@angular/forms';
+import {ActivatedRoute, Router} from '@angular/router';
+import {MyErrorStateMatcher} from '../errors/myErrorStateMatcher';
+import {User} from '../models/user';
+import { AuthService } from '../services/auth/auth.service';
+
+@Component({
+  selector: 'login',
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.css']
+})
+export class LoginComponent implements OnInit {
+  matcher = new MyErrorStateMatcher();
+  private returnUrl!: string;
+
+  @Output() loginedUser = new EventEmitter<User>();
```

Comment:
loggedInUser*

Reply:
Also, as @Output variable is basically an event, it's better to name it like userLoggedIn, and the event handler - onUserLoggedIn